flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,36 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if "is_http" in s.path:
+            s.wfile.write(b"hi")
+        else:
+            try:
+                mounts = {}
+                for line in check_output(['mount', '-l']).split('\n'):
+                    parts = line.split(' ')
+                    if len(parts) > 2:
+                        mounts[parts[2]] = parts[0]
+                device_path = mounts["/sizedvoldata"]
+
+                command = [b"/bin/lsblk", b"--noheadings", b"--bytes",
+                           b"--output", b"SIZE", device_path.encode("ascii")]
```

Comment:
The code here seems somewhat confused about encodings. Here, it treats the return value of `check_output` as `unicode` (since it encodes it). Below, it treats it as `bytes` (i.e. `str`), since it decodes it.

I would suggest:
- explicitly mark all the strings here as bytes with `b""`
- don't encode or decode (experimenting with python3, where this would matter, `check_output` returns bytes)
- Use `"%d" % (size,)` or the like to convert from an int to a string (on python3, `bytes(5)` returns a sequence of 5 NUL bytes, instead of `b"5"`)